src/features/Applet/Schedule/Calendar/Calendar.styles.ts

```diff
@@ -0,0 +1,118 @@
+import { styled, Box } from '@mui/material';
+
+import theme from 'styles/theme';
+import { variables } from 'styles/variables';
+import { StyledClearedButton } from 'styles/styledComponents/ClearedButton';
+
+export const StyledAddBtn = styled(StyledClearedButton)`
+  z-index: 4;
```

Comment:
better to move Z-index  to some global constants

Reply:
Done.